public/theme/boltcms-2021/newsitem.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main_container %}
+    <div class="detail-page">
+        {{ record|allow_twig }}
```

Comment:
Doing `{{ record|allow_twig }}` is not needed anymore (luckily!). This is a new improvement in Bolt itself. Let's remove it.

Reply:
Check